tests/BearSerializationTest.py

```diff
@@ -0,0 +1,34 @@
+import itertools
+import json
+import os
+import json
```

Comment:
You are importing json twice :)

On 12 Jul 2016 6:19 pm, "Sheikh Araf" <notifications@github.com> wrote:
>
> In tests/BearSerializationTest.py:
>
> > @@ -0,0 +1,34 @@
> > +import itertools
> > +import json
> > +import os
> > +import json
>
> json? It's used at L32
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.